tee-worker/core/tls-websocket-server/src/encrypt.rs

```diff
@@ -0,0 +1,87 @@
+use core::num::NonZeroU32;
+#[cfg(feature = "sgx")]
+use std::vec::Vec;
+
+use log::warn;
+use rsa::{Pkcs1v15Encrypt, RsaPublicKey};
+
+pub struct Encryptor {
+	key: RsaPublicKey,
```

Comment:
I remember we wanted to use AES key?

Reply:
AES is symmetric, so the key sent to the enclave has to be encrypted, while RSA does not need it.
Since the enclave is guaranteed to be real, I believe it safe to send a public key with clear text.